src/_includes/banner.html

```diff
@@ -1,9 +1,10 @@
 <div class="banner">
   <p class="banner__text">
-    Dart 3 is here with records and patterns, alongside Google I/O!
+    Dart 3 is here with 
```

Comment:
**nit:** Active voice and avoid here in text.
```suggestion
    Dart 3 released, featuring
```

Reply:
I'm so confused by active voice 😅 Wouldn't "released" beg the question, "_Who_ released Dart 3?", making "Dart 3 released" passive?